pkg/api/v1beta3/types.go

```diff
@@ -360,6 +364,30 @@ type GlusterfsVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty" description:"glusterfs volume to be mounted with read-only permissions"`
 }
 
+// RBDVolumeSource represents a Rados Block Device Mount that lasts the lifetime of a pod
+type RBDVolumeSource struct {
+	// Required: EndpointsName is the endpoint name that represents a collection of Ceph monitors
+	EndpointsName string `json:"endpoints" description:"a collection of Ceph monitors"`
+	// Optioanl: RBDPool is the rados pool name,default is rbd
+	RBDPool string `json:"pool" description:"rados pool name"`
+	// Required: RBDImage is the rados image name
+	RBDImage string `json:"image" description:"rados image name"`
+	// Optional: RBDUser is the rados user name, default is admin
+	RadosUser string `json:"user" description:"rados user name"`
```

Comment:
@thockin: Do we want to emphasize the 'Optional' part along with the default value in the json description? If we expect swagger to be the primary way for accessing API information, then I think the json description should indicate if a field is optional or not.